examples/roads2png.rs

```diff
@@ -1,23 +1,26 @@
 extern crate bresenham;
 extern crate failure;
 extern crate flatdata;
+extern crate itertools;
 extern crate osmflat;
 extern crate png;
+extern crate structopt;
 
 use bresenham::Bresenham;
 use failure::Error;
 use flatdata::{Archive, FileResourceStorage};
+use itertools::Itertools;
 use png::HasParameters;
+use structopt::StructOpt;
 
 use std::convert;
-use std::env;
 use std::fs::File;
 use std::io::BufWriter;
-use std::path::Path;
+use std::path::{Path, PathBuf};
 
 use std::str;
 
-#[derive(Debug, Clone, Copy, Default, PartialEq, PartialOrd)]
+#[derive(Debug, Clone, Copy)]
 struct GeoCoord {
     lat: f64,
     lon: f64,
```

Comment:
Could you please rename into `lng`? Should we also switch to f32?

Reply:
I would not rename, OSM uses lon, not lng

Reply:
Yes, I had it wrong in my head.